configure_hosts.yml

```diff
@@ -16,7 +16,7 @@
     - pdf_gen
   tasks:
     - name: get dns from local /etc/hosts
-      local_action: "command grep -E '^${% for group in groups.all %}|{{ '\\<%s\\>'|format(group) }}{% endfor %}' /etc/hosts"
+      local_action: "command grep -E '\\<{{ accounts_domain }}\\>{% for group in groups.all %}|{{ '\\<%s\\>'|format(group) }}{% endfor %}' /etc/hosts"
```

Comment:
It used to be:

```
grep -E "^$|\<local.cnx.org\>|\<zope.local.cnx.org\>|\<127.0.0.1\>" /etc/hosts
```

It is now:

```
grep -E "\<accounts.local.openstax.org\>|\<local.cnx.org\>|\<zope.local.cnx.org\>|\<127.0.0.1\>" /etc/hosts
```

getting the `/etc/hosts` entries for `local.cnx.org` (like in python `\blocal.cnx.org\b`) and `zope.local.cnx.org`.  Then I realized it's not looking for `accounts.local.openstax.org` which I needed in the VM.  Does that answer your question?